demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -85,11 +90,11 @@ def is_valid_file(self, validate_rn: bool = True, skip_test_conf: bool = False)
             self.is_valid_pwsh(),
             self.is_valid_image(),
             self.is_valid_description(beta_integration=False),
+            self.is_valid_max_fetch_and_first_fetch()
```

Comment:
so a line change won't pop in the next validation that will be added.
```suggestion
            self.is_valid_max_fetch_and_first_fetch(),
```